zkvm/spec/ZkVM.md

```diff
@@ -209,17 +209,17 @@ when these are exposed to the VM (for instance, from [`mul`](#mul)), they have t
 
 ### Attached and detached variables
 
-A [variable](#variable-type) can be in one of two states: **detached** and **attached**.
+A [variable](#variable-type) can be in one of two states: **detached** or **attached**.
 
-**Detached variable** can be [reblinded](#reblinded): all copies of a detached variable share the same commitment,
+A **Detached variable** can be [reblinded](#reblinded): all copies of a detached variable share the same commitment,
 so reblinding one of them reflects the new commitments in all the copies. When an [expression](#expression-type) is formed using detached variables, all of them transition to an _attached_ state.
 
-**Attached variable** has its commitment applied to the constraint system, so it cannot be reblinded and variable cannot be detached.
+An **Attached variable** has its commitment applied to the constraint system, so it cannot be reblinded and variable cannot be detached.
 
 
 ### Expression type
 
-_Expression_ is a linear combination of [variables](#variable-type) with cleartext [scalar](#scalar) weights.
+_Expression_ is a linear combination of attached [variables](#variable-type) with cleartext [scalar](#scalar) weights.
```

Comment:
I changed it to attached variables because in the detached variable section, it says that: "When an expression is formed using detached variables, all of them transition to an attached state."